.changeset/grumpy-meals-kick.md

```diff
@@ -0,0 +1,6 @@
+---
+'@web/test-runner-saucelabs': minor
+'@web/test-runner-webdriver': minor
```

Comment:
shouldn't other packages be updated too?
from the top of my head it's the browserstack one
https://github.com/modernweb-dev/web/blob/master/packages/test-runner-browserstack/package.json
maybe some others, good to check where else the `@web/test-runner-webdriver` is in deps

Reply:
Good catch, added `@web/test-runner-browserstack`. This is the only remaining package that needs updating (other packages only use webdriver launcher as dev dependency).